pkg/networkmanager/interfaces.js

```diff
@@ -2729,6 +3021,35 @@ PageNetworkInterface.prototype = {
                                 .syn_click(self.model, function () { configure() })));
             }
 
+            function render_general_settings_row() {
```

Comment:
In this function there are some strings that need to be translated (wrap them in `_("string")`.